### DISCUSSION TITLE
I need the GitHub support to remove 2FA from the account.

### DISCUSSION THREAD

**Discussion by sysop-88** [community/community#126922] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I lost my **2FA recovery** for my account. I changed my computer. I need **GitHub support** to remove **2FA** because it is locked out.

**Comment by CopperEagle** [comment]
Hey @sysop-88 

You may want to look at the [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials) on what you need to do in this case. There is also a virtual assistant that may help you there, right at the top of the page.

Also, before anything: Do you still have your one-time-use tokens (recovery code) somewhere? You can use those to log in and set up 2FA again. This is the best option. There are some more in the documentation.

If nothing else works, then you can start a **2FA recovery request**. This requires you to have access to your email account and will take several business days. The process is detailed [here](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#requesting-help-with-two-factor-authentication).

Also you can start a request for help [here](https://support.github.com/contact/account).

**Reply by sysop-88** [reply]
I made a support ticket [**2fa password recovery #2818086**](https://support.github.com/ticket/personal/0/2818086) which has not been replied for 8 days. I think something wrong with the support ticket. 
I need to remove **2FA** to log in since I lost my **2FA recovery**.

**Reply by sysop-88** [reply]
I also find many discussion on GitHub community about the support tickets have not been replied for long time. Is there anyway I can contact the GitHub support team about the support ticket?

**Comment by Ayoubbenderdouch** [comment]
how i can help you

**Comment by sysop-88** [comment]
I close this discussion and move to New User Help. **Moderator help**

**Comment by ebndev** [comment]
This post has been closed and locked because of version of it has already been posted and answered [here](https://github.com/orgs/community/discussions/127780#discussioncomment-9743453) by the Community Discussions Team.
